lib/theme_check/cli.rb

```diff
@@ -65,6 +66,10 @@ def option_parser(parser = OptionParser.new, help: true)
         "--print",
         "Output active config to STDOUT"
       ) { @command = :print }
+      @option_parser.on(
+        "--update-resources",
```

Comment:
Can I suggest `--update-docs`? `--update-resources` feels too much like an internal representation of the code. `--update-docs` might be easier to remember/understand for external folks.

Reply:
Sure! Thank you, @charlespwd :) I initially was trying to avoid the `docs` because the flag impacts the language server documentation and also some checks.

Still, users indeed might have a more general interpretation of the term `docs` (like, the may read it as Theme Check general docs).